### DISCUSSION TITLE
Integration Issue of Verso Wallet with WalletConnect mobile

### DISCUSSION THREAD

**Discussion by Fabrice-Fabio** [WalletConnect/walletconnect-monorepo#3916] [Developer Support]
Hello,

I am experiencing an issue with integrating Verso into dapps. While my wallet correctly appears in the explorer at https://walletconnect.com/explorer/verso when I search for my "Verso" wallet, it is not listed among the wallet options when trying to access WalletConnect from a mobile dapp.

Problem Details:

- From any mobile website of a dapp (example : app.aave.com)
- Click on wallet connect
- Many installed wallets are proposed but Verso is not in that list
- Wallet is present in the WalletConnect explorer : https://walletconnect.com/explorer/verso
- Attempts: Tried accessing from various mobile devices without success. (If an admin can contact me that would be great.)

Still our wallet is declared in wallet connect explorer

I would appreciate assistance in resolving this issue.

Thank you in advance for your help.

Best regards.

**Comment by glitch-txs** [comment]
which dapp are you testing this with?

Check here please, let me know if it's useful: https://docs.walletconnect.com/web3wallet/mobileLinking

**Comment by Fabrice-Fabio** [comment]
This has been tested on several dapps like uniswap, pancakeswap and many others.

The deeplink has already been set up and is available on "verso://verso.app" or for example "verso://verso.app/wc?uri=wc:94caa59c77dae0dd234b5818fb7292540d017b27d41f7f387ee75b22b9738c94@2?relay-protocol=irn&symKey=ce3 a2c7724c03cf1769ba8b1bdedad5414cc7b920aa3fb72112b997d1916266f"

**Reply by glitch-txs** [reply]
can you try also handling wc://

**Reply by Fabrice-Fabio** [reply]
Hello,
Thanks i tried this one "wc://" and it works well on android.
But on iOS, it doesn't work, any idea ?

**Reply by glitch-txs** [reply]
have you tried this?
https://docs.walletconnect.com/web3wallet/mobileLinking#ios-wallet-support

**Reply by Fabrice-Fabio** [reply]
I'll check that everything is okay and I'll get back to you.
but when doc said "examplewallet://wc?uri=wc:94caa59c77dae0dd234b5818fb7292540d017b27d41f7f387ee75b22b9738c94@2?relay-protocol=irn&symKey=ce3a2c7724c03cf1769ba8b1bdedad5414cc7b920aa3fb72112b997d1916266f"

examplewallet is the app name which i set in walletconnect backoffice or my app name ?

**Reply by glitch-txs** [reply]
examplewallet:// is the deeplink for the wallet app

**Reply by Fabrice-Fabio** [reply]
so i my case this link should work also on iOS for example right ?

wc://wc?uri=wc:94caa59c77dae0dd234b5818fb7292540d017b27d41f7f387ee75b22b9738c94@2?relay-protocol=irn&symKey=ce3a2c7724c03cf1769ba8b1bdedad5414cc7b920aa3fb72112b997d1916266f

**Reply by Fabrice-Fabio** [reply]
For example this code which calls the deeplink works well on iOS emulator: xcrun simctl openurl booted "wc://wc?uri=wc:470c0d544b298f7bcfc314df097380d0848f11057a0f44913e6d26c2c9c84d47@2?relay-protocol=irn&symKey=7d c9afa2c5c107b8cdf36600fc3bf5b746f9e095babf45a62ee2d92da21837ef"

On the other hand, once deployed, wallet connect neither finds the back wallet nor triggers it.

An idea ? because the configurations seem to be goodFor example this code which calls the deeplink works well on iOS emulator: xcrun simctl openurl booted "wc://wc?uri=wc:470c0d544b298f7bcfc314df097380d0848f11057a0f44913e6d26c2c9c84d47@2?relay-protocol=irn&symKey=7d c9afa2c5c107b8cdf36600fc3bf5b746f9e095babf45a62ee2d92da21837ef"

On the other hand, once deployed, wallet connect neither finds the back wallet nor triggers it.

An idea ? because the configurations seem to be good





![simulator_screenshot_3BC58B51-2EB3-4442-99D0-126AFD6CEDDC](https://github.com/WalletConnect/walletconnect-monorepo/assets/35635121/12704f64-3b93-4b09-954b-657475374d1a)

**Reply by glitch-txs** [reply]
Hey could you check if you have v2 enabled in the cloud form?

**Reply by Fabrice-Fabio** [reply]
Hey,
I just activated it and resubmitted it. It is in pending mode.
If there is a way to speed up the review process or if you could take a look at it, that would be great.
Thanks

**Reply by glitch-txs** [reply]
could you share your project ID?

**Reply by Fabrice-Fabio** [reply]
Sure, here is : 6277c824c6dda5702fca5a8bffc13181
